java/semantickernel-api/src/main/java/com/microsoft/semantickernel/contextvariables/converters/PrimitiveVariableContextVariableTypeConverter.java

```diff
@@ -34,11 +34,17 @@ public PrimitiveVariableContextVariableTypeConverter(
         super(
             clazz,
             s -> convert(s, clazz),
-            toPromptString,
+            PrimitiveVariableContextVariableTypeConverter.toPromptStringFunction(toPromptString),
```

Comment:
I hadn't noticed before, but the order of arguments here is different than in ContextVariableTypeConverter. All CVTC types should have the same order of args in the constructors. 

Can this be `s -> escapeXmlString(toPromptString.apply(s))` to make it parallel to `s -> convert(s, clazz)`